app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/PlanWizardInstancePropertiesStepActions.js

```diff
@@ -1 +1,31 @@
+import URI from 'urijs';
+import API from '../../../../../../../../common/API';
 
+import {
+  FETCH_V2V_OSP_SECURITY_GROUPS,
+  FETCH_V2V_OSP_FLAVORS
+} from './PlanWizardInstancePropertiesStepConstants';
+
+export const _getOpenstackSecurityGroupsActionCreator = url => dispatch => {
```

Comment:
change 
`export const _getOpenstackSecurityGroupsActionCreator = url => dispatch => {`
`to export const _getOpenstackSecurityGroupsActionCreator = url => dispatch => `

Reply:
Was just temporary for console.logs that I have since removed 🌮 🌮 🌮  but thank you for your attention to detail 🥇